cmd/influxd/launcher/launcher.go

```diff
@@ -269,29 +281,52 @@ func (m *Launcher) run(ctx context.Context, opts *InfluxdOpts) (err error) {
 
 	var flushers flushers
 	switch opts.StoreType {
-	case BoltStore:
-		store := bolt.NewKVStore(m.log.With(zap.String("service", "kvstore-bolt")), opts.BoltPath)
-		store.WithDB(m.boltClient.DB())
-		m.kvStore = store
+	case DiskStore, BoltStore:
+		kvStore := bolt.NewKVStore(m.log.With(zap.String("service", "kvstore-bolt")), opts.BoltPath)
+		kvStore.WithDB(m.boltClient.DB())
+		m.kvStore = kvStore
+
+		// If a bolt path is specified and the sqlite path has NOT been specified, put the
+		// sqlite data in the same folder as the bolt file.
+		dir, _ := fs.InfluxDir()
+		if opts.BoltPath != filepath.Join(dir, bolt.DefaultFilename) &&
+			opts.SqLitePath == filepath.Join(dir, sqlite.DefaultFilename) {
+			opts.SqLitePath = filepath.Dir(opts.BoltPath) + "/" + sqlite.DefaultFilename
```

Comment:
I like this, but wonder wonder if it will be confusing to users, since the generated help-text for the `--sqlite-path` flag is going to show `~/.influxdbv2`. We should look into ways to make the help-text more explanatory.